data/json/items/gun/223.json

```diff
@@ -1386,5 +1386,75 @@
       }
     ],
     "melee_damage": { "bash": 10 }
+  },
+  {
+    "id": "rdb_223",
+    "copy-from": "rifle_semi",
+    "looks_like": "ar15",
+    "type": "GUN",
+    "name": { "str": "5.56x45mm bullpup rifle" },
+    "description": "This is a semi-automatic bullpup rifle with a lightened body due to the use of polymers and aluminum, using standard magazines of the AR-15 platform and a 5.56x45mm cartridge.",
+    "variant_type": "gun",
+    "variants": [
+      {
+        "id": "rdb_223",
+        "name": { "str_sp": "Kel-Tec RDB" },
+        "description": "This is a semi-automatic bullpup rifle with a lightened body due to the use of polymers and aluminum, using standard magazines from the AR-15 platform and the 5.56 NATO cartridge.  The name stands for Right Done Bullpup due to the incorporation of technical solutions, such as downward case ejection and convenience for left-handers, designed to solve some of the problems of bullpup guns.  However, the good idea's execution is not as perfect as the manufacturer (and user) would like.",
```

Comment:
RDB stands for "rifle, downward-ejection, bullpup". 
The last sentence could use some fluffing, too, IMO. "No good idea from Kel-Tec is immune to Kel-Tec manufacturing issues, sadly."